src/stores/DataStore.js

```diff
@@ -94,7 +95,7 @@ export const DataStore = Fluxxor.createStore({
         const { datetimeSelection, timespanType } = this.dataStore;
         const { defaultLanguage, logo, title, targetBbox, supportedLanguages, defaultZoomLevel } = configuration;
         const { fromDate, toDate } = convertDateValueToRange(datetimeSelection, timespanType);
-
+        
```

Comment:
Nit: trailing spaces. If you use `git add -p` to stage your changes incrementally, it'll highlight trailing spaces in big ugly red to alert you to them. After any `git add` on the command line, git will also tell you about "x lines add whitespace errors" if you have trailing spaces.

In case you're using the vim plugin for vscode, you can automatically get rid of all trailing spaces with this command: `:%s/ *$//g`